manifests/init.pp

```diff
@@ -27,13 +29,24 @@
   $config             = {},
   $manage_persistence = false,
   $slaveof            = undef,
+  $version            = 'redis-3.0.5-1.el7.cmm.x86_64',
```

Comment:
This is not an appropriate default for a public module.  The default should be `installed`.